### ISSUE TITLE
LDraw Packing script doesn't mention you also need to create a package.json to run

### ISSUE THREAD

**Issue by DennisSmolek** [mrdoob/three.js#28862]
### Description

When trying to load some new models with the lDrawLoader it points to the packing script:
https://github.com/mrdoob/three.js/blob/master/utils/packLDrawModel.js

This has instructions to run it. 
It fails.

Adding a package.json fixes the problem:
```json
{
    "name": "ldraw-model-packer",
    "version": "1.0.0",
    "description": "A script to pack LDraw model files.",
    "type": "module",
    "scripts": {
        "start": "node packLDrawModel.js"
    },
    "author": "",
    "license": "ISC"
}
```

### Reproduction steps

1. Follow the instructions.
2. Console errors
3. generate package.json
4. works

### Version

166

**Comment by Mugen87** [comment]
> It fails.

Do you mind sharing the error? Looking at the script, I don't understand how the addition of a `package.json` solves any issue.

**Comment by DennisSmolek** [comment]
So this may totally be my ignorance with setting up Node to run scripts outside an application. but it complains about loading modules and says to set the type to module in package.json (which clears the error) or use .mjs (which does not)

```sh
PS C:\Users\Public\Documents\LDraw> node packLDrawModel models/bonsai.mpd
(node:21004) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
C:\Users\Public\Documents\LDraw\packLDrawModel.js:23
import fs from 'fs';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1153:20)
    at Module._compile (node:internal/modules/cjs/loader:1205:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1295:10)
    at Module.load (node:internal/modules/cjs/loader:1091:32)
    at Module._load (node:internal/modules/cjs/loader:938:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)
    at node:internal/main/run_main_module:23:47

Node.js v20.9.0
```

**Comment by DennisSmolek** [comment]
The instructions in the file say:
```ts
* Usage:
 *
 * - Download official parts library from LDraw.org and unzip in a directory (e.g. ldraw/)
 *
 * - Download your desired model file and place in the ldraw/models/ subfolder.
 *
 * - Place this script also in ldraw/
 *
 * - Issue command 'node packLDrawModel models/<modelFileName>'
 *
 * The packed object will be in ldraw/models/<modelFileName>_Packed.mpd and will contain all the object subtree as embedded files.
 *
 ```

This fails and I couldn't see a setup without a package.json that worked

**Comment by Mugen87** [comment]
How about renaming `packLDrawModel.js` to `packLDrawModel.mjs` instead?

**Comment by Mugen87** [comment]
Should be fixed via c483bccc7e9082f7da0ae5bdb9131ae843dd9ea2
